src/librustdoc/html/static/js/main.js

```diff
@@ -963,7 +963,7 @@ function preLoadCss(cssUrl) {
     }
 
     onEachLazy(document.getElementsByClassName("tooltip"), e => {
-        e.onclick = function() {
+        e.onclick = () => {
             this.TOOLTIP_FORCE_VISIBLE = this.TOOLTIP_FORCE_VISIBLE ? false : true;
```

Comment:
pretty sure `this` isn't available in closures

Reply:
Very good point. Should have used `e`.